tests/models/test_dagrun.py

```diff
@@ -2009,3 +2009,41 @@ def execute(self, context):
         ti.run()
 
     assert sorted(results) == expected
+
+
+def test_mapped_task_group_expands(dag_maker, session):
+    with dag_maker(session=session):
+
+        @task_group
+        def tg(x, y):
+            return MockOperator(task_id="task_2", arg1=x, arg2=y)
```

Comment:
@uranusjr do you know if it is possible to inspect any of the Mapped Arguments' fields within the task group here at runtime? I'm trying to and I get a type error saying MappedArgument is not subscriptable when trying to index into a dictionary arg

Reply:
You need to be more specific about what you want. Also better to open a discussion please.